doc/content/devices/mac-settings/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "MAC Settings"
+description: ""
+---
+
+This section provides guidelines for configuring MAC settings for end devices from the CLI.
+
+<!--more-->
+
+{{< cli-only >}}
+
+### MAC Settings and MAC State
+
+`mac-settings` on {{% tts %}} are configurable per end device. To configure persistent MAC settings, make changes to `mac-settings.desired_<parameter>`. Updates to `mac-settings.desired_<parameter>` take effect on device creation, after OTAA join or ABP FCnt reset, ResetInd, or after MAC state reset.
+
+`mac-state` can be used to test MAC settings in the current session. To update settings for testing in the current session, make changes to the `mac-state.desired_parameters.<parameter>`. Updates to the `mac-state.desired_parameters.<parameter>` are applied on the next uplink, and lost on reset.
+
+The expected procedure for testing and updating settings is:
+
+1. Modify `mac-state.desired_parameters.<parameter>` to see changes in the current session
+2. Test that everything works as expected
+3. Modify `mac-settings.desired_<parameter>` to make the change permanent
+
+If no settings are provided on device creation or unset, defaults are taken from [Network Server Configuration]({{< ref src="/reference/configuration/network-server" >}}).
```

Comment:
Or frequency plan of the device.

By order of preference:
User overrides (MAC settings) -> Frequency Plan (and regional spec) -> Network Server configuration